docs/dev/infrastructure.rst

```diff
@@ -1,86 +1,208 @@
 ********************************
-JDAViz Design and Infrastructure
+Jdaviz Design and Infrastructure
 ********************************
 
-This document outlines the top-level structure of jdaviz. At the highest level, jdaviz layers different, sometimes changing technologies in the Jupyter platform to do its visualization, and therefore provides a framework for these technologies to work together.  This document describes that framework, as well as the high-level sets of components needed for the jdaviz use cases at the time of this document's writing. Bear in mind, though, that the fast moving scientific and technical fields jdaviz is aimed at mean that some of the specific details in this document may change faster than the document can be kept up, so the high-level structure is the primary focus of this document, with the specific components best thought of as *examples* rather than a comprehensive list of all components implemented.  (For that, see the documentation itself).
+.. note::
 
-This document lists the layers of the `jdaviz` framework in essentially the order in which they contact users in a typical visualization-heavy workflow.  These layers are: interfaces, applications, component widgets, and libraries. An overview of the layers is in this diagram, and each is described in more detail below:
+    At the time of writing, Jdaviz was still in heavy development.
+    If you notice that this page has fallen behind, please
+    `report an issue to the Jdaviz GitHub issues <https://github.com/spacetelescope/jdaviz/issues/new>`_.
 
-.. image:: jdaviz.svg
+This section outlines the top-level structure of Jdaviz. At the highest level,
+Jdaviz layers different, sometimes changing technologies in the Jupyter platform
+to do its visualization, and therefore provides a framework for these technologies
+to work together. This document describes that framework, as well as the high-level
+sets of components needed for the Jdaviz use cases; it lists the layers of the Jdaviz
+framework in essentially the order in which they contact users in a typical
+visualization-heavy workflow. An overview of the layers is in this diagram,
+and each is described in more details below:
 
-Interfaces
-==========
+.. figure:: jdaviz.svg
+    :alt: Jdaviz design and infrastructure chart.
 
-"Interfaces" are the tool the user is using to access the analysis tools. The word "platform" might at first seem more applicable, but in this case all of the interfaces are using Jupyter as the platform, to ensure a consistent look-and-feel and a single platform for which to target the tools.  The interfaces are then specific interfaces through which users access this platform.  The target interfaces are:
+    This figure illustrates the basics of Jdaviz design and infrastructure.
```

Comment:
@orifox -- This would be the figure caption that also goes in the paper. Suggestion welcome.